helm-chart/templates/deployment.yaml

```
@@ -38,7 +38,10 @@ spec:
         - configMapRef:
             name: {{ include "json-rpc-relay.fullname" . }}
         - secretRef:
-            name: {{ include "json-rpc-relay.fullname" . }}  
+            name: {{ include "json-rpc-relay.fullname" . }}
+        {{- with .Values.extraEnvFrom }}
```

Also not sure if you need the with but could also simplify with below. Again from mirror node
```suggestion
        envFrom: {{ tpl (toYaml .Values.envFrom) . | nindent 12 }}
```

---

Based on above I guess this also doesn't apply.